pkg/OWNERS

```diff
@@ -1,11 +1,13 @@
 reviewers:
   - brendandburns
+  - davidopp
   - dchen1107
   - lavalamp
   - smarterclayton
   - thockin
 approvers:
   - brendandburns
+  - davidopp
```

Comment:
New top level owner?

Reply:
Result of a somewhat naive merge. I can push this one into pkg/scheduler/OWNERS.